python/pylibvw.cc

```diff
@@ -54,6 +57,113 @@ const size_t pDECISION_SCORES = 8;
 
 void dont_delete_me(void* arg) {}
 
+class OptionManager
+{
+  std::map<std::string, std::vector<VW::config::option_group_definition>> m_option_group_dic;
+  py::object m_cla;
```

Comment:
what does `cla` stand for?

Reply:
class, this is where I store the python class that is used to contain each vw option. It should be renamed to something more descriptive

Reply:
done